<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>student</class>
 <widget class="QMainWindow" name="student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_top">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1026</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_background">
     <property name="geometry">
      <rect>
       <x>445</x>
       <y>0</y>
       <width>601</width>
       <height>65</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_background">
     <property name="geometry">
      <rect>
       <x>452</x>
       <y>65</y>
       <width>574</width>
       <height>35</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="button_manaRoll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>学籍管理</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_innovation">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>0</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>创新项目</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_infosearch">
      <property name="geometry">
       <rect>
        <x>236</x>
        <y>0</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>信息查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_more">
      <property name="geometry">
       <rect>
        <x>354</x>
        <y>0</y>
        <width>120</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_background_2">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>10</y>
       <width>239</width>
       <height>53</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>frame_background</zorder>
    <zorder>label_background</zorder>
    <zorder>label_background_2</zorder>
   </widget>
   <widget class="QFrame" name="frame_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>141</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="button_infor_search">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>信息查询</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_grade_search">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>32</y>
       <width>141</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>成绩查询</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_roll_search">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>64</y>
       <width>141</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>学籍查询</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_main">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>99</y>
      <width>891</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="size_putton">
     <property name="geometry">
      <rect>
       <x>843</x>
       <y>0</y>
       <width>46</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="putton_delete">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>0</y>
       <width>46</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="main_frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>891</width>
       <height>35</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_desktop">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>  我的桌面</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_innovation">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>  创新项目</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_grade_search">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>  成绩查询</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="table_grade">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>70</y>
       <width>891</width>
       <height>35</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>申报人编号</string>
      </property>
      <property name="background">
       <color alpha="0">
        <red>241</red>
        <green>241</green>
        <blue>241</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>申报人姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>课题编号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>课程名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>批次名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>课题简介</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>指导教师一</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="table_grade_2">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>105</y>
       <width>891</width>
       <height>35</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>3214002920</string>
      </property>
      <property name="background">
       <color alpha="0">
        <red>241</red>
        <green>241</green>
        <blue>241</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>张小轩</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>A1111</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>流行音乐研究</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2018春季</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>暂无</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>陈奕迅</string>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="main_frame_pro">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>891</width>
       <height>35</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_apply_pro">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>  申报项目</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_search_pro">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>  已选课题</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_update_pro">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>  修改项目</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="table_grade_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>140</y>
       <width>891</width>
       <height>35</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>指导教师二</string>
      </property>
      <property name="background">
       <color alpha="0">
        <red>241</red>
        <green>241</green>
        <blue>241</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>教师联系方式一</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>选题开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>选题结束时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>申报资金</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>审核状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="table_grade_4">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>170</y>
       <width>891</width>
       <height>331</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>周杰伦</string>
      </property>
      <property name="background">
       <color alpha="0">
        <red>241</red>
        <green>241</green>
        <blue>241</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>13387987892</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2018.1.1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2019.1.1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>10000</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>受审</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>暂无</string>
      </property>
     </column>
    </widget>
   </widget>
   <zorder>frame_top</zorder>
   <zorder>frame_main</zorder>
   <zorder>frame_left</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
